docs_user/assemblies/assembly_migrating-ceph-rgw.adoc

```diff
@@ -6,12 +6,12 @@ ifdef::context[:parent-context: {context}]
 
 = Migrating {Ceph} RGW to external RHEL nodes
 
-For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running {Ceph} version 6 or later, you must migrate the RGW daemons that are included in the {rhos_prev_long} Controller nodes into the existing external Red Hat Enterprise Linux (RHEL) nodes. The existing external RHEL nodes typically include the Compute nodes for an HCI environment or {Ceph} nodes.
+For Hyperconverged Infrastructure (HCI) or dedicated Storage nodes that are running {Ceph} version 6 or later, you must migrate the Ceph Object Gateway (RGW) daemons that are included in the {rhos_prev_long} Controller nodes into the existing external Red Hat Enterprise Linux (RHEL) nodes. The external RHEL nodes typically include the Compute nodes for an HCI environment or {Ceph} nodes.
 
-To migrate Ceph Object Gateway (RGW), your environment must meet the following requirements:
+To migrate RGW, your environment must meet the following requirements:
 
-* {Ceph} is running version 6 or later and is managed by cephadm/orchestrator.
-* An undercloud is still available, and the nodes and networks are managed by {OpenStackPreviousInstaller}.
+* {Ceph} is running version 6 or later and is managed by `cephadm` or Ceph Orchestrator.
+* The undercloud is still available, and the nodes and networks are managed by {OpenStackPreviousInstaller}.
```

Comment:
`The undercloud is still available, and the nodes and networks are managed by {OpenStackPreviousInstaller}.`

@klgill I think the above is an old requirement. and it's about the fact we might want to propagate any potential missing network to the target nodes before going through the migration. I propose to remove this sentence and eventually put a link (later in the procedure) to the `Prerequisites` section.
We did a similar conversation for `mon` migration, and the issue is that having the Ceph procedure at the end of the adoption process means that we might not have an undercloud/overcloud available anymore, so it doesn't make sense put this `prerequisite` here.
A better idea could be moving it at the very beginning of the the whole adoption guide, in the form: "...and if you have Ceph, make sure to complete the following prerequisites using tripleo".

Reply:
@fmount  Ah yes, I remember this conversation now. As you said, for FR1, we should develop a complete list of prerequisites that Ceph users must complete before they attempt the Ceph migration (or start any of the adoption procedure). 
I'll hide the "Undercloud" prerequisite and add a comment that we need to include it in the prerequisites in the first chapter.

Reply:
I created a Jira card so we don't lose track: https://issues.redhat.com/browse/RHOSPDOC-2024